wpninjas/ninja-forms-uploads
# Use Dropbox API v2

Resolves https://github.com/wpninjas/ninja-forms-uploads/issues/261

- [x] Upgrade oauth token v1 > v2
- [x] Test on PHP 5.2

_Release_

- [x] Change Dropbox API Client app to use redirect uri https://oauth.polevaultweb.com/


**Testing**

- Whilst on `master` disconnect Dropbox if test install already connected.
- Login to your Dropbox personal account, go to settings, security and revoke the NF FU app
- Switch to branch 
- Try to authorize, cancel when Dropbox asks you to authorize - see error notice back in WP
- Authorize successfully
- Create form with FU and external action for dropbox
- Test for submission, file uploaded to dropbox
- Test download of FU in submissions screen
- Disconnect from Dropbox in wp
- Switch back to `master` authorize Dropbox in wp (oauth1)
- Switch to branch and try to submit the form (oauth1 token should be migrated to v2) and submission to dropbox should still work.
